workflows/genomeassembly.nf

```diff
@@ -20,6 +20,11 @@ if (params.cool_bin) { cool_bin = params.cool_bin } else { cool_bin = 1000; }
 if (params.polishing_on) { polishing_on = params.polishing_on } else { polishing_on = false; }
 if (params.hifiasm_hic_on) { hifiasm_hic_on = params.hifiasm_hic_on } else { hifiasm_hic_on = false; }
 if (params.organelles_on) { organelles_on = params.organelles_on } else { organelles_on = false; }
+
+// Declare constants to toggle BUSCO for alts
+set_busco_alts = true
+unset_busco_alts = false
```

Comment:
These two variables is just opposite each other, keep two or you can merge into one?

Reply:
The two variables were intended to increase readability at the point of usage :
```
GENOME_STATISTICS_RAW_HIC( ...., set_busco_alts)
GENOME_STATISTICS_PURGED( ...., unset_busco_alts)
```
If leave only <code>set_busco_alts</code> then it will look like

```
GENOME_STATISTICS_RAW_HIC( ...., set_busco_alts)
GENOME_STATISTICS_PURGED( ...., not set_busco_alts)
```

Or if we keep the value set/unset in the pipeline by default the usage will look like

```
GENOME_STATISTICS_RAW_HIC( ...., set_busco_alts)
GENOME_STATISTICS_PURGED( .... )
```

I think I prefer the first option. Although it would be ideal to set them in constants. Do you know if it's possible to do in nextflow?